src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1984,6 +1984,7 @@
   </Node>
 
   <Node Name="BoundArrayInitialization" Base="BoundExpression">
+    <Field Name="IsInferred" Type="bool"/>
```

Comment:
>IsInferred [](http://example.com/codeflow?start=17&length=10)

Consider adding a comment. Also, I think we generally following an unspoken convention that "Type" is the first property in the node description.